docs/reference/inference/post-inference.asciidoc

```diff
@@ -64,30 +65,43 @@ seconds.
 
 [discrete]
 [[post-inference-api-request-body]]
-== {api-request-body-title}
+==== {api-request-body-title}
 
 `input`::
-(Required, array of strings)
+(Required, string or array of strings)
 The text on which you want to perform the {infer} task.
 `input` can be a single string or an array.
++
+--
 [NOTE]
 ====
-Inference endpoints for the `completion` task type currently only support a single string as input.
+Inference endpoints for the `completion` task type currently only support a
```

Comment:
I think we should at Tim Grein to review the completion components here https://github.com/elastic/elasticsearch/pull/106476

Reply:
He was the one who originally added the docs here: https://github.com/elastic/elasticsearch/pull/106876
The only change I've done here is some change of the markup.